src/inputs/Select/Select.js

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React, { useMemo, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { Stack, Autocomplete, TextField } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+import { TooltipInfo } from '../../misc';
+import { BasicInputPlaceholder } from '../BasicInputs/BasicInputPlaceholder';
+import { getCommonInputStyles } from '../style';
+
+const useStyles = makeStyles(getCommonInputStyles);
+
+export const Select = (props) => {
+  const classes = useStyles();
+  const { id, labels, tooltipText, value, options, disabled, onChange, isDirty } = props;
+  const getLabel = useCallback((key) => options?.find((el) => el.key === key)?.label, [options]);
+
+  const optionValue = useMemo(() => {
+    return { key: value, label: getLabel(value) };
+  }, [getLabel, value]);
+
+  if (disabled) {
+    const selectedValues = optionValue.label ?? labels.noValues;
```

Comment:
There can be only one value selected in this component: could we rename `selectedValues` to `selectedValue`?